Procfile

```diff
@@ -1 +1 @@
-web: ROCKET_PORT=$PORT ROCKET_ENV=prod ./target/release/rustbridge
+web: ROCKET_PORT=$PORT ROCKET_ENV=prod ./rustbridge/target/release/rustbridge
```

Comment:
this is surprising to me... if it works i'm all for it, but i am curious why this is needed?